stack.yaml

```
@@ -27,6 +27,7 @@ extra-deps:
   - micro-recursion-schemes-5.0.2.2
   - streaming-binary-0.3.0.1
   - canonical-json-0.6.0.0
+  - base64-0.4.1
```

```suggestion
  - base64-0.4.2
  - ghc-byteorder-4.11.0.0.10
```
I needed to insert these new extra-deps to compile it using `stack`

---

Ah possible. I've compiled 1.13.0, master has probably already diverged.

---

Actually no, it requires `base64-0.4.1` also on latest master.